process_creationg_termination.c

```diff
@@ -16,19 +15,17 @@ int main()
     childPid = fork();
     if(childPid  >= 0)
     {
-        if(childPid == 0)
-        {
-            printf("Child process\n");
-            sleep(5);
+        if(childPid == 0) {
+            printf("Child greeting\n");
+            sleep(2);
             printf("Child exiting\n");
         }
-        else
-        {
-            waitpid(-1,NULL,0);
+        else{
+           sleep(2);
```

Comment:
yes they do, thats the reason we use WAIT. so that the parent waits till the child terminates. sleep would just delay it from execution for that many seconds. where as wait would specifically wait till the child terminates.